nouveau/TODO

```diff
@@ -0,0 +1,27 @@
+targeted dreyfus feature parity
+
+* pagination (bookmark I guess)
+* grouping
+* faceting
+* partitioned db support
+* ken integration
```

Comment:
This seems to be at odds with the README which says ken integration is supported.

What about just making the TODO a part of README?. It's developer documentation anyway, users would use look official CouchDB docs most likely.

Reply:
yup, out of date.